Update WooCommerce Services extensions banner

v18 opened the pull request:
Changes in this PR

- The banner image was updated
- Responsive styles were added
- Banner text was updated

I used your copy suggestion, @MarinaWoo, but I didn't mention the parts about tracking shipping for now. cc @MarinaWoo for another look :)

## Here's what it looks like now on on the featured page:

<img width="941" alt="featured-page-big-screen" src="https://cloud.githubusercontent.com/assets/11487924/26745324/fcb757d4-47b7-11e7-9617-c78f744aed61.png">

## and on the shipping page:

<img width="1528" alt="shipping-page-big-screen-2" src="https://cloud.githubusercontent.com/assets/11487924/26745343/10b0112c-47b8-11e7-8f2c-da5a0efa9472.png">

## the shipping page on a slightly smaller screen:

<img width="946" alt="shipping-page-big-screen-1" src="https://cloud.githubusercontent.com/assets/11487924/26745342/10af9ad0-47b8-11e7-8ba2-6963f29a6607.png">

I know it doesn't quite line up with the other boxes on that page, but that would quite a bit of work. If there's a better look that doesn't rely on the width of the boxes, I'm all ears :)

## On a small screen for featured:

<img width="396" alt="500px-featured" src="https://cloud.githubusercontent.com/assets/11487924/26745369/35409124-47b8-11e7-9a4c-f0efeca9097f.png">

## On a small screen for shipping:

<img width="397" alt="500px-shipping" src="https://cloud.githubusercontent.com/assets/11487924/26745370/3545266c-47b8-11e7-88c3-30d9cea0ed31.png">

## Here's a mid-way point for reference:
<img width="596" alt="medium-screen-featured" src="https://cloud.githubusercontent.com/assets/11487924/26745397/4b2c370e-47b8-11e7-9890-0296d5563934.png">

@olaolusoga any final thoughts? :) cc @jeffstieler @nabsul @garymurray

v18 committed:
Update WCS banner text

v18 committed:
Update WCS banner image

mikejolley commented on includes/admin/class-wc-admin-addons.php:
```diff
@@ -332,30 +332,32 @@ public static function output_wcs_banner_block( $block = null ) {
 			'image_alt'   => __( 'WooCommerce Services', 'woocommerce' ),
-			'title'       => __( 'Transform your store into a shipping and fulfillment machine', 'woocommerce' ),
-			'description' => __( 'WooCommerce Services makes shipping a breeze. Print a label, take advantage of discounted shipping rates, and send tracking information to your customer right as you process your order, all from the convenience of your WordPress dashboard.', 'woocommerce' ),
+			'title'       => __( 'Buy discounted shipping labels, print them from your dashboard.' ),
```
Missing textdomain

mikejolley replied:
Should this be:

> Buy discounted shipping labels AND print them from your dashboard.

v18 replied:
Fixed

v18 replied:
> Should this be:

> Buy discounted shipping labels AND print them from your dashboard.

I think it _could_ be that. cc @olaolusoga and @MarinaWoo for word-smithing input.

mikejolley commented on includes/admin/class-wc-admin-addons.php:
```diff
@@ -332,30 +332,32 @@ public static function output_wcs_banner_block( $block = null ) {
-			'title'       => __( 'Transform your store into a shipping and fulfillment machine', 'woocommerce' ),
-			'description' => __( 'WooCommerce Services makes shipping a breeze. Print a label, take advantage of discounted shipping rates, and send tracking information to your customer right as you process your order, all from the convenience of your WordPress dashboard.', 'woocommerce' ),
+			'title'       => __( 'Buy discounted shipping labels, print them from your dashboard.' ),
+			'description' => __( 'WooCommerce Services makes shipping a breeze. Get discounted rates, and print labels – all from the convenience of your WordPress dashboard.', 'woocommerce' ),
```
No comma before 'and'

v18 replied:
Fixed

mikejolley reviewed (changes requested):
Few changes needed. Can you sync with master too? May need to update npm dependencies so it doesn't minify everything.

v18 committed:
Update WCS banner style

v18 commented:
Fixed the issues above, and rebased.

> Can you sync with master too? May need to update npm dependencies so it doesn't minify everything.

Sure, I rebased on master and ran `grunt css` again.
